packages/client/src/pages/Track/track.tsx

```diff
@@ -0,0 +1,34 @@
+import { Date } from '@components/Date.component';
+import { DisplayCard } from '@components/DisplayCard.component';
+import { DisplayData } from '@pages/Track/displaydata';
+import { ProjectTable } from '@components/table/ProjectTable';
+import styled from 'styled-components';
+
+export const Track = () => {
+  const TrackContainer = styled.div`
+    width: 100%;
+    height: 10rem;
+    display: flex;
+    flex-direction: column;
+    gap: 4rem;
+    align-items: start;
+  `;
+
+  const DisplayContainer = styled.div`
+    display: flex;
+    gap: 4rem;
+    align-items: center;
+  `;
+
+  return (
+    <TrackContainer>
+      <DisplayContainer>
+        <Date />
+        {DisplayData.map((data) => {
+          return <DisplayCard key={data.id} item={data} />;
```

Comment:
since there are only two or three cards, it is better to just list them out. If props change will have to look thru the array and rerender all cards. If props change will only have to rerender the cards were props change